app.rb

```diff
@@ -70,6 +70,22 @@ def set_locale(locale)
     redirect "/"
   end
 
+  put '/rewind' do
+    game = session[:game]
+    if game.board.unplayed?(game.current_state)
+        redirect "/"
+    end
+    if game.inactive_player.class == Computer
+        game.go_back(2)
+    else
+        game.go_back(1)
+    end
+    
+    if request.xhr?
```

Comment:
is this necessary? this looks like its for handling http requests outside the application.

Reply:
oh nevermind i just saw that this is for handling requests from your javascript code. so my next question is should this be two separate endpoints? because its able to return two different responses correct? we can talk more on this if you have questions.

Reply:
@TakaGoto Hey can we talk over slack tomorrow about this issue in particular? I'd like to ask you some questions